WordPress/Sniffs/PHP/NoSilencedErrorsSniff.php

```diff
@@ -36,33 +36,35 @@ class NoSilencedErrorsSniff extends Sniff {
 	public $context_length = 6;
 
 	/**
-	 * Whether or not the `$function_whitelist` should be used.
+	 * Whether or not the `$allowedFunctionsList` should be used.
 	 *
 	 * Defaults to true.
 	 *
-	 * This property only affects whether the standard function whitelist is
-	 * used. The custom whitelist, if set, will always be respected.
+	 * This property only affects whether the standard function list is
+	 * used. The custom functions list, if set, will always be respected.
 	 *
 	 * @since 1.1.0
+	 * @since 3.0.0 Renamed from `$use_default_whitelist` to `$usePHPFunctionsList`.
 	 *
 	 * @var bool
 	 */
-	public $use_default_whitelist = true;
+	public $usePHPFunctionsList = true;
 
 	/**
-	 * User defined whitelist.
+	 * User defined function list.
 	 *
-	 * Allows users to pass a list of additional functions to whitelist
-	 * from their custom ruleset.
+	 * Allows users to pass a list of additional functions for which to allow
+	 * the use of the silence operator. This list can be set in a custom ruleset.
 	 *
 	 * @since 1.1.0
+	 * @since 3.0.0 Renamed from `$custom_whitelist` to `$customFunctionsList`.
 	 *
 	 * @var array
 	 */
-	public $custom_whitelist = array();
+	public $customFunctionsList = array();
```

Comment:
☝🏻 This one.

This is a `public` property, so manipulable from a custom ruleset.

I wonder if `$customFunctionsList` isn't too vague and whether this should be changed to `$customAllowedFunctionsList` ?

Reply:
`$customAllowedFunctionsList` sounds clearer, so I'd go with that 👍🏼

Reply:
I've fixed this up in the original commit.